ci-operator/step-registry/hypershift/conformance/hypershift-conformance-chain.yaml

```diff
@@ -95,7 +95,19 @@ chain:
       openshift-tests run ${TEST_SUITE} ${TEST_ARGS:-} \
         --provider "${TEST_PROVIDER}" \
         -o "${ARTIFACT_DIR}/e2e.log" \
-        --junit-dir "${ARTIFACT_DIR}/junit"
+        --junit-dir "${ARTIFACT_DIR}/junit" 2>&1 | tee /tmp/openshift-tests.log
+      
+      exit_code=$?
+                
+      # If this line is shown in the openshift-tests output, all of the conformance test cases passed,
+      # but the monitor test has failed. Currently we're ignoring MonitorTest failures until we resolve:
+      # https://issues.redhat.com/browse/CNV-33717
+      if [[ -n "${SKIP_MONITOR_TEST}" ]] && [[ grep "failed due to a MonitorTest failure" /tmp/openshift-tests.log ]]; then
+        exit_code=0
```

Comment:
also, can you print here something like

`echo "overriding Monitor Test Failure" ` so we'll have something that indicates clearly when thsi occurs.